docker-tests/tests/docker_tests/test_docker_compose_quick_start.py

```diff
@@ -115,6 +115,10 @@ def test_trigger_dag_and_wait_for_result(default_docker_image, tmp_path_factory,
         # Before we proceed, let's make sure our DAG has been parsed
         compose.execute(service="airflow-dag-processor", command=["airflow", "dags", "reserialize"])
 
+        # Verify API server health endpoint is accessible and returns valid response
+        health_status = api_request("GET", path=f"monitor/health").get("metadatabase").get("status")
```

Comment:
Oh, just a nit in static checks, can you fix?
```suggestion
        health_status = api_request("GET", path="monitor/health").get("metadatabase").get("status")
```